examples/taco/nodejs/src/index.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  conditions,
+  decrypt,
+  encrypt,
+  fromBytes,
+  getPorterUri,
+  initialize,
+  toBytes,
+} from '@nucypher/taco';
+import * as dotenv from 'dotenv';
+import { ethers } from 'ethers';
+
+dotenv.config();
+
+const rpcProviderUrl = process.env.RPC_PROVIDER_URL;
+if (!rpcProviderUrl) {
+  throw new Error('RPC_PROVIDER_URL is not set.');
+}
+
+const privateKey = process.env.PRIVATE_KEY;
+if (!privateKey) {
+  throw new Error('PRIVATE_KEY is not set.');
```

Comment:
We have a hardcoded example key for Enrico in `nucypher`. See https://github.com/nucypher/nucypher/blob/4b978025a262f240a9afb9869257c070a33d767d/tests/constants.py#L88

Maybe we can use this here as a fallback?

Reply:
On one hand, it makes sense - This Enrico is already configured on testnet (allow-listed and so on) but on the other hand, I'm not sure what happens when a number of users start testing their apps with this default character.

Reply:
I don't think we have much of a choice for `tapir` 🤔 , unless:
1. we make ritual initiation public (for say tapir)
2. grant initiator roles to those who ask

only specific Enricos would be allowed (global allow list) to encrypt for rituals.

If neither 1 or 2 are feasible, at the very least we can probably provide more than one test authorized Enrico account private key.

Reply:
Decided to not pursue this change in the PR, tracking it here: https://github.com/nucypher/nucypher-ts/issues/313